src/Worker.js

```diff
@@ -53,15 +54,21 @@ function prepareJscodeshift(options) {
 
 function setup(tr, babel) {
   if (babel === 'babel') {
-    require('babel-register')({
+    require('@babel/register')({
       babelrc: false,
       presets: [
-        require('babel-preset-es2015'),
-        require('babel-preset-stage-1'),
```

Comment:
`@babel/preset-stage-1` is deprecated. See https://babeljs.io/docs/en/babel-preset-stage-1
It was added in https://github.com/facebook/jscodeshift/commit/28eb51508dca2aa70f9a454d3247ee2e08fcf4ae.
Are there any experimental features we'd like to include?

Reply:
Good question. I'll take a look at the current proposal.

Another question though: Can `@babel/preset-flow` and `@babel/preset-typescript` be both loaded at the same time? It feels like this should be incompatible (at least you cannot set both options in babylon).
cc @hzoo

Reply:
Yeah, you shouldn't be able to because they would both try to set options in babylon and fail as incompat

Reply:
Oh interesting. Thank you so much for pointing this out! I didn't get an error from babel register about it and it appeared to run fine on the TS file I tested. But, I realize that my test TS file was valid flow so I didn't see any issues. I tried adding a TypeScript-only construct and it does indeed fail. I updated this PR to only load the typescript preset if it's a `.tsx?` transform.